app/assets/stylesheets/sequence_builder.css

```diff
@@ -209,8 +202,40 @@ span.prefix{
   font-weight: bold;
 }
 
-/* SITE-WIDE ISH */
+/* SITE-WIDE BACKGROUND */
 
 body.ng-scope {
 background: #EEE;
+}
+
+/* SITE WIDE STUFF */
+
+.green-button{
```

Comment:
Is this different than the preexisting `green` class? Will this class supersede the current `green` class that is already implemented, or is it unrelated? If it's going to supersede it, let me know so I can clean up what's left.